deeprank/learn/NeuralNet.py

```diff
@@ -991,6 +972,7 @@ def _plot_scatter_reg(self, figname):
         ax.set_ylabel('Predictions')
 
         values = np.append(xvalues, yvalues)
+
         border = 0.1 * (values.max() - values.min())
         ax.plot([values.min() - border, values.max() + border],
                 [values.min() - border, values.max() + border])
```

Comment:
Functions _plot_hit_rate _compute_hitrate and _get_relevance use iRMSD values that we do not have.
I think we can ask Li about usability of these functions.

Reply:
**about plot_hit_rate:** this method is called in the "NeuralNet.test" function, that we do not call

**about compute_hitrate**: this is commented out in deeprank's code

**about get_relevance**: we don't use this in our version. The developers adviced me to set "NeuralNet.save_hitrate" to False, after getting an error message.

Reply:
Thanks Coos.
I still find 'compute_hitrate' and 'get_relevance' included in the current version on NeuralNet.py. 
Is there another version where these are not used?

Reply:
Indeed the _compute_hitrate method is declared, but it's not used anywhere. We might as well remove it. The original call of this function is commented out at line 719.
For the get_relevance method it's about the same story. It's declared, but not used, since we set save_hitrate to False.

Do you want me to remove the two methods completely?

Reply:
I think plotting is something we need to check once we have all the data from Bioprodict. The plots would make sense when you have a dataset of variants. 
Plotting functions can be retained. The unused _compute_hitrate and _get_relevance could be removed? These functions also use target_ordering

Reply:
I removed the two methods in this commit: https://github.com/DeepRank/deeprank/commit/01d0f9f8fdd7a93754063b496bbeb5a603917efc I left the hitrate plotting method in, though we cannot use it right now.